src/ServiceControl.Plugin.Nsb6.Heartbeat/Heartbeats.cs

```diff
@@ -116,24 +119,24 @@ void SendEndpointStartupMessage(HostInformation hostInfo, DateTime startupTime,
                 {
                     Logger.Warn(string.Format("Unable to register endpoint startup with ServiceControl. Going to reattempt registration after {0}.", registrationRetryInterval), ex);
 
-                    Task.Delay(registrationRetryInterval, cancellationToken)
-                        .ContinueWith(t => SendEndpointStartupMessage(hostInfo, startupTime, cancellationToken), cancellationToken);
+                    await Task.Delay(registrationRetryInterval, cancellationToken).ConfigureAwait(false);
+                    await SendEndpointStartupMessage(startupTime, cancellationToken).ConfigureAwait(false);
                 }
             }
 
-            void SendHeartbeatMessage(HostInformation hostInfo)
+            async void SendHeartbeatMessage()
```

Comment:
Shhhhhhh

![you didnt see anything](https://media.giphy.com/media/XItRQJP0wai7m/giphy.gif)